api/src/services/__tests__/programsService.ts

```diff
@@ -168,6 +170,16 @@ const participantActivitiesList: ParticipantActivities[] = [
     completed: true,
   },
 ];
+
+const participantActivitiesForProgram: participantAcitivityForProgram = {
+  programId: 1,
+  participantActivities: [
+    { activity_id: 7, completed: false },
```

Comment:
Perhaps these values can be extracted from `participantActivitiesList` instead of re-declaring them?